sdk/eventgrid/azure-eventgrid/azure/eventgrid/_helpers.py

```diff
@@ -165,3 +165,30 @@ def _build_request(endpoint, content_type, events):
     )
     request.format_parameters(query_parameters)
     return request
+
+def _get_json_content(obj):
+    """Event mixin to have methods that are common to different Event types
+    like CloudEvent, EventGridEvent etc.
+    """
+    try:
+        # storage queue
+        return json.loads(obj.content)
+    except ValueError:
+        raise ValueError(
+            "Failed to retrieve content from the object. Make sure the "
+                + "content follows the EventGridEvent schema."
```

Comment:
We can probably rephrase this now that it lives in a json-specific context - maybe something like:
"Failed to load JSON content from the object."